go/ephemeral/errors.go

```diff
@@ -0,0 +1,94 @@
+package ephemeral
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/gregor1"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+type EKType string
+
+const (
+	DeviceEKStr EKType = "deviceEK"
+	UserEKStr   EKType = "userEK"
+	TeamEKStr   EKType = "teamEK"
+)
+
+type EphemeralKeyError struct {
+	DebugMsg string
+	HumanMsg string
+}
+
+const (
+	defaultHumanErr                             = "This exploding message is not available to you"
```

Comment:
cc @malgorithms / @cecileboucheron, any opinion on the copy?

Reply:
which of the unavailable scenarios does this apply to?

Reply:
this pr covers:

 - the user provisioned the device since the message was sent
 - the user cloned a device

that particular line is just the default fallback if we have no specific info

Reply:
error you have seems fine